src/com/docker/projects/Documentation.groovy

```diff
@@ -0,0 +1,51 @@
+
+
+package com.docker.projects;
+
+// Add the following code to any Jenkinsfile.
+// It will return with success if there are no changes in the specified docs dir
+// and Will return with success if run on a non Pull Request branch
+
+// def docs = new com.docker.projects.Documentation()
+// docs.documentationPRChecker("docs")
+
+// docsDir ~~ the subdirectory within your repo that the documentation resides
+def documentationPRChecker(docsDir) {
+  if (env.CHANGE_ID != null) {
+    stage "docs PR checker"
+
+    tokens = "${env.JOB_NAME}".tokenize('/')
+    org = tokens[0]
+    repo = tokens[1]
+    branch = tokens[2]
+    sh "git log --format='%H' -1 > tempoutput"
+    sha = readFile("tempoutput").trim()
+    imageName = "${repo}:${branch}"
+    containerName = "${repo}-${branch}-${env.BUILD_ID}"
```

Comment:
yeah, i stopped using BUILD_TAG when I noticed it contains the randomly made pipeline name

```
BUILD_TAG=jenkins-Documentation pipeline-docs.docker.com-v1.12-12
```